mods/webui/src/stories/link/modaltrigger/ModalTrigger.styles.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2024 by Fonoster Inc (https://fonoster.com)
+ * http://github.com/fonoster/fonoster
+ *
+ * This file is part of Fonoster
+ *
+ * Licensed under the MIT License (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    https://opensource.org/licenses/MIT
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { styled } from "@mui/material/styles";
+
+export const StyledModalTrigger = styled("button")(({ theme }) => ({
```

Comment:
Can you have another pass at the fidelity from Figma? For example, I noticed that the font family is yet to be set. And the font size is set to `16px` instead of `10px`.

<img width="1019" alt="Developer Mode for component" src="https://github.com/user-attachments/assets/2d3341ab-a1e5-4f0c-9ae3-9fa510f0ca8b" />